pnpm-lock.yaml

```diff
@@ -418,7 +418,7 @@ importers:
       '@openzeppelin/contracts': 3.4.1
       '@popperjs/core': 2.9.2
       '@servie/events': 3.0.0
-      '@snapshot-labs/snapshot.js': github.com/snapshot-labs/snapshot.js/f32bf0f5b4a27a23b0db3c6fdaced1abab25f55a
+      '@snapshot-labs/snapshot.js': github.com/snapshot-labs/snapshot.js/e2a6ac28560f22030104b39bb983294a0a3cb73d
```

Comment:
@Jack-Works i think need lock snapshot.js version
CC @guanbinrui

Reply:
@zhouhanseng suggest, revert this pnpm-lock changed

Reply:
It's locked by commit hash.

Reply:
https://github.com/DimensionDev/Maskbook/blob/7fe73abd542ae7271fcd3c16cac747efcb6d02d9/packages/maskbook/package.json#L35

Reply:
https://github.com/DimensionDev/Maskbook/pull/3467#pullrequestreview-689599022

Reply:
I think this pkg should not be updated in this PR. We need to lock the version of `@snapshot-labs/snapshot.js` in the `package.json` file.

Reply:
https://github.com/DimensionDev/Maskbook/pull/3490
plz review lock snapshot.js pr